probe/main.go

```diff
@@ -153,3 +176,16 @@ func interrupt() chan os.Signal {
 	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
 	return c
 }
+
+func interfaces() string {
```

Comment:
I think it is a sensible default, or at least I can't think of a way to produce a better default. Would you suggest something like only take `{en,eth}[0-9]*` as a default? Note that for the Docker entrypoint, we can change it via commandline flag...